src/test/resources/epub-indexes/indexes-package-document.feature

```diff
@@ -0,0 +1,53 @@
+Feature: EPUB Indexes Package Document
+  
+  Checks conformance to rules for Package Documents defined in the
+  EPUB Indexes specification:
+    
+    http://idpf.org/epub/idx/
+  
+  This feature file contains tests for EPUBCheck running in `opf` mode to check
+  single Package Documents (`.opf` files).
+  
+  Note: 
+  - Tests related to EPUB Indexes package rules in a full EPUB publication
+    are defined in the `indexes-package.feature` feature file.
+  - Tests related to general conformance of Package Documents are defined
+    in the `epub3/package-document.feature` feature file.
+
+  Background: 
+    Given EPUB test files located at '/epub-indexes/files/package-document/'
+    And EPUBCheck configured to check a Package Document
+
```

Comment:
Don't you have to configure for the edupub profile, too?

Reply:
No, I don't think so. As far as I understand and remember, these tests are for index collections that can be part of any EPUB, but not specifically for index-only EPUBs. So the rules are always triggered as part of the default check.

This may require some double-check, as my memories tend to fail me 😅. For one thing, the `OPFCheckerTests.java` code that was ported didn't specify the index profile.

Reply:
Ah, okay. I stole the file for the xhtml content documents tests and noticed it didn't have it but it is needed for each test.

Reply:
Right, for XHTML the profile would need to be explicitly set to trigger the checks. When not sure, you can look at the Java code, if the profile appears explicitly in the test method.